lib/services/ngsi/ngsiService.js

```diff
@@ -490,7 +490,7 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
                     }
 
                     options.json = {
-                        actionType: 'APPEND',
+                        actionType: 'append',
```

Comment:
Maybe a line about this changes should be added to CHANGES_NEXT_RELEASE (e.g. `Fix: using "append" instead of "APPEND" (which is deprecated) in NGSIv2 operations`).

Reply:
Fixed with 8b32fb06be08d6b118d11e66f101d3508d1de01d